### ISSUE TITLE
ScreenReader: map settings button not visible in chrome

### ISSUE THREAD

**Issue by danielw97** [Trimps/bug-tracker#225]
**Describe the bug**

When playing Trimps with a screenreader (NVDA 2023.1 in my case) on Windows 11 inside of Google Chrome the map settings button isn't visible.
It is  visible in Firefox, however.

**To Reproduce**

1. Navigate to trimps.github.io/ScreenReader in Chrome whilst using nvda.
2. Upon reaching Zone 6, notice that the map settings button isn't visible to nvda as it was previously.

**Expected behavior**

The map settings button should be visible to NVDA as it was previously.

**Screenshots**

**Environment (please complete the following information):**
 - OS version (Windows 10, 11? Linux?)
 - Windows 11
 - Game Version (are you playing on the web, or on Steam)?
Web
 - If you're playing on the web, which browser are you using? [e.g. Chrome, Firefox, Safari]
Chrome

**Additional context**
I will mention that as previously stated I am using a screenreader (NVDA) to access the game.
This is a problem that only happens in chrome, not in firefox.
I would switch over to firefox, however it doesn't appear that tooltips etc are read out correctly.
Thanks in advance for looking into this, and let me know if there's any more information I can provide.

**Comment by ahicks92** [comment]
The bug is that the map is using title on a span.  It should probably be aria-label.  Looks like Chrome fixed this on their end, arguably.
